### PR TITLE
Adding method allow_migrate on MasterSlaveRouter

### PR DESCRIPTION
**Pull request by raghuraju**
Prevents it from breaking with Django 1.8.*

### CODE REVIEW

**Comment by raghuraju** [comment]
Can anybody please review these changes and do the needful?

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='django-multidb-router',
-    version='0.6',
+    version='0.7',
```
Comment (jsocol) [inline setup.py]: please don't bump the version in patches, those are handled in separate commits

File: multidb/__init__.py [hunk 2]
```diff
@@ -79,6 +79,11 @@ def allow_syncdb(self, db, model):
         """Only allow syncdb on the master."""
         return db == DEFAULT_DB_ALIAS
 
+    def allow_migrate(self, db, app_label, model_name=None, **hints):
+        """Determine if the migration operation is allowed to run on the database with alias db."""
```
Comment (jsocol) [inline multidb/__init__.py]: these comments are definitely >80 chars. Mind rewording to be shorter?

Comment (raghuraju) [inline multidb/__init__.py]: Sure, will do.

File: setup.py (hunk 1 above)
Comment (raghuraju) [inline setup.py]: Sorry, this was to force pip to install my changes as I had no control over the server.

**Comment by diox** [comment]
Thanks for this, I had not seen that before submitting #32. Going to close this one because of the unnecessary version bump in `setup.py` that has not been fixed since the initial review.

**Pull request closed without merging**